Week3/homework/step4.js

```diff
@@ -2,7 +2,14 @@
 
 // Add your code here
 
-const addSix = createBase(6);
 
-addSix(10); // returns 16
-addSix(21); // returns 27
+function sumTen(num) {
```

Comment:
Aahaan. I think you slightly missed the point of the exercise. You need to first provide a base addition as a parameter to a function. That function would then return another function to which you can provide a new value as 10 or 21. For example, in this case, our function only adds 6 to a number. What if I want to add 60 instead. 😉 

Remember closures? Let me know if you still have any confusion.

Reply:
@yash-kapila  yupe, I could use more of your knowledge because.. Yeah, I Remember closures but to be honest not as much as I want to !

Reply:
Okay. So a simple way to remember closures is that `if an inner function remembers the variables from its outer function even when the outer function has returned is called a closure`. 

Now, in our case, we want our outer function to accept a base argument which is going to be 6 for now. And from this function, we would want to return another function which accepts a second argument and returns the sum of that and the outer function's parameter.

I would suggest checking the example of closures which we discussed in the project I had shared.